libraries/chain/controller.cpp

```diff
@@ -3993,80 +3992,78 @@ struct controller_impl {
       auto qc_ext_id = quorum_certificate_extension::extension_id();
       bool qc_extension_present = block_exts.count(qc_ext_id) != 0;
 
-      EOS_ASSERT( !qc_extension_present, invalid_qc_claim,
+      EOS_ASSERT( !qc_extension_present, block_validate_exception,
```

Comment:
For consistency, this should either remain `invalid_qc_claim` or we should change similar checks involving the integrity of the QC block extension in `verify_proper_block_exts`.

Reply:
For now I reverted back. I will clean up exception types in `verify_proper_block_exts` in future PR.